testing/test_assertrewrite.py

```diff
@@ -70,10 +70,18 @@ class TestAssertionRewrite(object):
     def test_place_initial_imports(self):
         s = """'Doc string'\nother = stuff"""
         m = rewrite(s)
-        # Module docstrings in 3.7 are part of Module node, it's not in the body
+        # Module docstrings in 3.8 are part of Module node, it's not in the body
         # so we remove it so the following body items have the same indexes on
         # all Python versions
-        if sys.version_info < (3, 7):
+        # This was briefly in 3.7 as well but got reverted in beta 5
+        # TODO:
+        # We have a complicated sys.version_info if in here to ease testing on
+        # various Python 3.7 versions, but we should remove the 3.7 check after
+        # 3.7 is released as stable
+        if (
```

Comment:
Could you move the check to its own function, moving the explanatory comment to its docstring? This will make the code easier to follow.

Reply:
yes